tests/virt_autotest/prepare_transactional_server.pm

```diff
@@ -50,7 +50,7 @@ sub prepare_extensions {
     my $self = shift;
 
     #Subscribing packagehub that enables access to many useful software tools
-    virt_autotest::utils::subscribe_extensions_and_modules(reg_exts => 'PackageHub');
+    virt_autotest::utils::subscribe_extensions_and_modules(reg_exts => 'PackageHub', activate => 0);
```

Comment:
Is it correct that the calling to virt_autotest::utils::subscribe_extensions_and_modules is not needed at all at the moment?  How about commenting this line?

Reply:
How about just keep it and use activate => 0 ?  So I do not need to do more verification runs. I think it does no harm.

Reply:
No big deal of course. I thought commenting off would be more efficient -- no jumping in and out of the function. But up to you to decide.

Reply:
Done